cmd/juju/config/config_test.go

```diff
@@ -152,6 +156,51 @@ func (s *suite) TestInitSuccess(c *gc.C) {
 	}
 }
 
+var fileContents = []byte(`
+key1: val1
+key2: val2`)
+var configAttrs = Attrs{
+	"key1": "val1",
+	"key2": "val2",
+}
+
+func (s *suite) TestReadFile(c *gc.C) {
```

Comment:
would be good to add one/two more tests that test bad yaml and non-existing file returns an error